UPGRADE-3.x.md

```diff
@@ -7,6 +7,10 @@ This interface has been deprecated without replacement.
 
 `ModelManagerInterface::getDefaultSortValues()` won't be used anymore.
 
+### Empty values in datagrid filters
+
+Now empty values are passed to datagrid filters. If you have custom datagrid filters, add empty string check to them.
+
```

Comment:
@VincentLanglet please review

Reply:
Adding an exemple could be helpful:

Something like

In the following exemple
```
->add('with_open_comments', CallbackFilter::class, [
    'callback' => static function(ProxyQueryInterface $queryBuilder, string $alias, string $field, array $value): bool {
        if (!$value['value']) {
            return false;
        }

        $queryBuilder
            ->leftJoin(sprintf('%s.comments', $alias), 'c')
            ->andWhere('c.status = :status')
            ->setParameter('status', Comment::STATUS_MODERATE);

         return true;
    },
    'field_type' => CheckboxType::class
 ]);
```
The `!$value['value']` check is required to not filtering by `''` if you didn't used the filter.

Reply:
I add this